Remotion/Core/Reflection/PropertyInfoAdapter.cs

```diff
@@ -232,7 +234,7 @@ public IEnumerable<IPropertyInformation> FindInterfaceDeclarations ()
       return _interfaceDeclarations.Value.ToArray();
     }
 
-    private IReadOnlyCollection<IPropertyInformation> FindInterfaceDeclarationsImplementation ()
+    private IReadOnlyCollection<IPropertyInformation?> FindInterfaceDeclarationsImplementation ()

```

Comment:
not nullable

Reply:
FindDeclaringProperty may return null per interface definition

Reply:
Hmm...I.C. 

Still wrong. If this were to occur, it would be a bug. It's not clear if the reflection implementation cannot trigger this scenario in the first place of if such members should be filtered. Fo now, we fix thsi with an NFO in the implementation and add a bugfix todo.